### DISCUSSION TITLE
Geoserver on GCP - cloudrun

### DISCUSSION THREAD

**Discussion by Solverj** [kartoza/docker-geoserver#443] [Q&A] [answered]
Hey, I'm having issues with: 
![image](https://user-images.githubusercontent.com/2386621/184641393-df4e7568-e150-49e5-952c-f4b6399ca3a8.png)

Everytime I try to save a workspace or save any form really. The only thing that works is logging inn, and that due to me having to add PROXY_BASE_URL
I'm a bit stuck on what seems to be CORS issues, and I have tried: GEOSERVER_CSRF_DISABLED and export GEOSERVER_CSRF_WHITELIST

Can you guys identify what the problem might be from the image provided?

**Comment by Solverj** [comment, marked as answer]
I figured it out
Set GEOSERVER_CSRF_WHITELIST=example.org, example.org:8080

**Reply by NyakudyaA** [reply]
Thanks, We might need to either document this in .env or in the readme

**Reply by jeff-bradley** [reply]
Hi @Solverj I am trying to use Cloud Run also and I am seeing the same issue, where did you enter the GEOSERVER_CSRF_WHITELIST values?

**Reply by NyakudyaA** [reply]
That should be captured as an environment variable
